### ISSUE TITLE
camera image hovering displays the image off screen

### ISSUE THREAD

**Issue by Cdetviler** [aprsworld/wsWebDisplay#72]
Bug has only been produced in chrome so far

**Comment by Cdetviler** [comment]
I reproduced the issue in chrome. It seems to behave differently depending on where the image sits on the page. Hovering is handled differently for webkit browsers and I have attempted to implement it in a way that accommodates the difference. I will have to alter the class that is added to the hovered image when using a webkit browser. A possible fix would be to change the css so that the image that pops up when hovering is fixed rather than absolute.

**Comment by Cdetviler** [comment]
I currently have it so that both Firefox and Chrome display in the upper right hand corner in order to stop the image from going off screen. This works for now, but a better solution might be desirable.

**Comment by Cdetviler** [comment]
With talk of adding more to the pop-out images (such as a playable movie), it seems that this feature will be used more in the future. With that in mind, I decided to devote a bit of time today to fix this issue. I changed the way the positioning is calculated, and it seems to be working a lot better now. I tested in:

Windows: IE, Chrome, Firefox
Linux: Chrome, Chromium, Firefox

I also found that if you hold your finger over an image in Android, the pop-out image behavior results. So I tested in the android browser as well.
